tools/unix/clone_en_release_notes.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ANDROID_LISTINGS=android/src/fdroid/play/listings
+ANDROID_LISTINGS=android/app/src/main/fdroid/play/listings
```

Comment:
Can fdroid be kept in android/app/src?

Reply:
We can but I don't really understand the issue